cmd/bnscli/cmd_termdeposit.go

```diff
@@ -139,7 +139,7 @@ This functionality is intended to extend UpdateConfigurationMsg message.
 	}
 	var (
 		periodFl = fl.Duration("period", 10*24*time.Hour, "Lockin period required for this bonus.")
-		bonusFl  = fl.Uint("bonus", 50, "Bonus value in percentage. Value 50 represents 50%.")
+		bonusFl  = flFraction(fl, "bonus", "1/2", "Bonus value in percentage. Value 50 represents 50%.")
```

Comment:
The hint needs to be changed now that we use a fraction.

Reply:
Great catch!